src/main/java/org/folio/ed/handler/ResponseHandler.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.ed.handler;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.integration.handler.GenericHandler;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ResponseHandler implements GenericHandler<String> {
+  private static final Logger LOGGER = LoggerFactory.getLogger(ResponseHandler.class);
+
+  @Override
+  public Object handle(String payload, MessageHeaders messageHeaders) {
+    LOGGER.info("Response handler income: {}", payload);
+    return null;
```

Comment:
Seems do nothing, could ResponseHandler be removed?

Reply:
According to the specification, we must receive the response from the server. Since `outboundGateway` requires handling of the response, but we have nothing to do with it, this handler terminates the flow correctly. If this handler will be removed, an exception will be thrown upon each primary channel exchange.